INFO     cardano_node_tests.utils.artifacts:artifacts.py:30 Coverage file saved to '/home/martink/Source/repos/cardano-node-tests7/.cli_coverage'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:30 Coverage file saved to '/home/martink/Source/repos/cardano-node-tests7/.cli_coverage'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:44 Start script coverage log file saved to '/home/martink/Source/repos/cardano-node-tests7/.cli_coverage/cli_coverage_script_220807_025153549_zmcj.log'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:80 Cluster artifacts saved to '/home/martink/Source/repos/cardano-node-tests7/tmp/pytest-of-martink/pytest-5/cluster_artifacts/state-cluster0_taiijdhj'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:44 Start script coverage log file saved to '/home/martink/Source/repos/cardano-node-tests7/.cli_coverage/cli_coverage_script_220807_025155156_yssc.log'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:80 Cluster artifacts saved to '/home/martink/Source/repos/cardano-node-tests7/tmp/pytest-of-martink/pytest-5/cluster_artifacts/state-cluster1_rnrfhjwj'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:44 Start script coverage log file saved to '/home/martink/Source/repos/cardano-node-tests7/.cli_coverage/cli_coverage_script_220807_025156647_cmfl.log'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:80 Cluster artifacts saved to '/home/martink/Source/repos/cardano-node-tests7/tmp/pytest-of-martink/pytest-5/cluster_artifacts/state-cluster2_cyfsmgxj'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:44 Start script coverage log file saved to '/home/martink/Source/repos/cardano-node-tests7/.cli_coverage/cli_coverage_script_220807_025158132_frtl.log'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:80 Cluster artifacts saved to '/home/martink/Source/repos/cardano-node-tests7/tmp/pytest-of-martink/pytest-5/cluster_artifacts/state-cluster3_xqrerauo'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:44 Start script coverage log file saved to '/home/martink/Source/repos/cardano-node-tests7/.cli_coverage/cli_coverage_script_220807_025159634_lwbp.log'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:80 Cluster artifacts saved to '/home/martink/Source/repos/cardano-node-tests7/tmp/pytest-of-martink/pytest-5/cluster_artifacts/state-cluster4_ijwvqhgf'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:44 Start script coverage log file saved to '/home/martink/Source/repos/cardano-node-tests7/.cli_coverage/cli_coverage_script_220807_025201074_aghq.log'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:80 Cluster artifacts saved to '/home/martink/Source/repos/cardano-node-tests7/tmp/pytest-of-martink/pytest-5/cluster_artifacts/state-cluster5_vfvrzrxg'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:44 Start script coverage log file saved to '/home/martink/Source/repos/cardano-node-tests7/.cli_coverage/cli_coverage_script_220807_025202532_lldv.log'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:80 Cluster artifacts saved to '/home/martink/Source/repos/cardano-node-tests7/tmp/pytest-of-martink/pytest-5/cluster_artifacts/state-cluster6_fgloiyng'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:44 Start script coverage log file saved to '/home/martink/Source/repos/cardano-node-tests7/.cli_coverage/cli_coverage_script_220807_025204049_kicd.log'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:80 Cluster artifacts saved to '/home/martink/Source/repos/cardano-node-tests7/tmp/pytest-of-martink/pytest-5/cluster_artifacts/state-cluster7_sxvcsthz'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:44 Start script coverage log file saved to '/home/martink/Source/repos/cardano-node-tests7/.cli_coverage/cli_coverage_script_220807_025205470_ngqh.log'.
INFO     cardano_node_tests.utils.artifacts:artifacts.py:80 Cluster artifacts saved to '/home/martink/Source/repos/cardano-node-tests7/tmp/pytest-of-martink/pytest-5/cluster_artifacts/state-cluster8_povxiqhl'.
INFO     cardano_node_tests.utils.dbsync_conn:dbsync_conn.py:34 Closing connection to db-sync database dbsync0.